instruments/thorlabs/thorlabsapt.py

```diff
@@ -1337,6 +1337,131 @@ def backlash_correction(self, pos):
             )
             self._apt.sendpacket(pkt)
 
+        @property
+        def home_parameters(self):
+            """Get the home parameters for the motor channel.
+
+            Parameters are stage specific and not all parameters can be set
+            for every stage. For example, the MLS203 stage only allows the
+            homing velocity to be changed.
+
+            .. note:: When setting the quantity, pass `None` to values
+                that you want to leave unchanged (see example below).
```

Comment:
I thought this would be neat for the setter. For my current application, I don't care about the velocity or which way around the limit switch works, I just want to leave that at default and only want to adjust the offset. This way, I can pass `None` for values I don't want to change and set parameters that I want.